repo/content/index/info.go

```diff
@@ -92,3 +93,32 @@ func ToInfoStruct(i Info) *InfoStruct {
 		EncryptionKeyID:     i.GetEncryptionKeyID(),
 	}
 }
+
+func DiffInfo(i0, i1 Info) []string {
+	var qs []string
+	if i0.GetFormatVersion() != i1.GetFormatVersion() {
+		qs = append(qs, fmt.Sprintf("mixed FormatVersions: %v %v", i0.GetFormatVersion(), i1.GetFormatVersion()))
+	}
+	if i0.GetOriginalLength() != i1.GetOriginalLength() {
+		qs = append(qs, fmt.Sprintf("mixed OriginalLengths: %v %v", i0.GetOriginalLength(), i1.GetOriginalLength()))
+	}
+	if i0.GetPackBlobID() != i1.GetPackBlobID() {
+		qs = append(qs, fmt.Sprintf("mixed PackBlobIDs: %v %v", i0.GetPackBlobID(), i1.GetPackBlobID()))
+	}
+	if i0.GetPackedLength() != i1.GetPackedLength() {
+		qs = append(qs, fmt.Sprintf("mixed PackedLengths: %v %v", i0.GetPackedLength(), i1.GetPackedLength()))
+	}
+	if i0.GetPackOffset() != i1.GetPackOffset() {
+		qs = append(qs, fmt.Sprintf("mixed PackOffsets: %v %v", i0.GetPackOffset(), i1.GetPackOffset()))
+	}
+	if i0.GetEncryptionKeyID() != i1.GetEncryptionKeyID() {
+		qs = append(qs, fmt.Sprintf("mixed EncryptionKeyIDs: %v %v", i0.GetEncryptionKeyID(), i1.GetEncryptionKeyID()))
+	}
+	if i0.GetDeleted() != i1.GetDeleted() {
+		qs = append(qs, fmt.Sprintf("mixed Deleted flags: %v %v", i0.GetDeleted(), i1.GetDeleted()))
+	}
+	if i0.GetTimestampSeconds() != i1.GetTimestampSeconds() {
+		qs = append(qs, fmt.Sprintf("mixed TimestampSeconds: %v %v", i0.GetTimestampSeconds(), i1.GetTimestampSeconds()))
+	}
+	return qs
```

Comment:
@adowair the above was taken out in an earlier version.  There was a bad merge that re-introduced it.

The code has been removed because its more sophisticated that what was required for the use-case.